tools/ci_build/github/azure-pipelines/linux-x86-ci-pipeline.yml

```
@@ -0,0 +1,40 @@
+jobs:
+- job: Linux_CI_Dev
+  pool: Linux-CPU
+  steps:
+    - template: templates/set-test-data-variables-step.yml
+
+    - task: CmdLine@2
+      displayName: 'Clean untagged docker images'
+      inputs:
+        script: |
+          docker rm $(docker ps -a | grep Exited | awk '{print $1;}') || true
+          docker images -q --filter "dangling=true" | xargs -n1 -r docker rmi
```

Consider using `prune` command here.

```suggestion
          docker container prune -f
          docker image prune -f
```

---

updated to use prune instead of --filter "dangling=true"